modules/va_facilities/spec/requests/nearby_request_spec.rb

```diff
@@ -1,420 +1,99 @@
-        get base_query_path + empty_address, params: nil, headers: accept_geojson
-        expect(response).to be_successful
-        json = JSON.parse(response.body)
-        expect(json['type']).to eq('FeatureCollection')
-        expect(json['features'].length).to eq(0)
-        expect(response.headers['Link']).to be_present
-        parsed = parse_link_header(response.headers['Link'])
-        expect(parsed['self']['page']).to eq(['1'])
-        expect(parsed['first']['page']).to eq(['1'])
-        expect(parsed['last']['page']).to eq(['1'])
+        expect(json['data'].length).to eq(1)
+        expect(nearby_result['attributes']['min_time']).to eq(0)
+        expect(nearby_result['attributes']['max_time']).to eq(10)
+        expect(nearby_result['id']).to eq('vha_648')
+        expect(nearby_result['relationships']['va_facility']['links']['related'])
+          .to eql('/services/va_facilities/v0/facilities/vha_648')
+        expect(json['meta']['pagination'])
+          .to include('current_page' => 1, 'per_page' => 20, 'total_pages' => 1, 'total_entries' => 1)
+        expect(json['links']['related']).to eq('/services/va_facilities/v0/facilities?ids=vha_648')
       end
     end
 
-    it 'responds with wait times as part of health services in v1' do
-      setup_pdx
-      VCR.use_cassette('bing/isochrone/pdx_drive_time_60',
-                       match_requests_on: [:method, VCR.request_matchers.uri_without_param(:key)]) do
-        get base_query_path + address_params, params: nil, headers: accept_geojson
+    it 'can be retrieved with a lat/lng' do
+      create_bands
 
-        expect(response).to be_successful
-        json = JSON.parse(response.body)
-        health_service = json['features'][0]['properties']['services']['health'][0]
-
-        expect(health_service['service']).to eq('PrimaryCare')
-        expect(health_service['wait_times']['new']).to eq(27.0)
-        expect(health_service['wait_times']['established']).to eq(6.0)
-        expect(health_service['wait_times']['effective_date']).to eq('2018-03-05')
-      end
-    end
-
-    it 'responds successfully even if health facilities return no health services' do
-      create 'vha_402QA'
-      VCR.use_cassette('bing/isochrone/no_health_services_drive_time',
-                       match_requests_on: [:method, VCR.request_matchers.uri_without_param(:key)]) do
-        get base_query_path + no_health, params: nil, headers: accept_json
-        expect(response).to be_successful
-
-        json = JSON.parse(response.body)
-        health_services = json['data'][0]['attributes']['services']['health']
-        expect(health_services.length).to eq(0)
-      end
-    end
-  end
+      get base_query_path,
+          params: { lat: 45.4967668, lng: -122.6832211, drive_time: '10' },
+          headers: { 'HTTP_ACCEPT' => 'application/json' }
 
-  context 'with invalid request parameters' do
```

Comment:
Why were these unhappy-path tests for param validations removed? Don't we need them?

Reply:
My mistake. Took them out when doing the overhaul to reduce noise. Adding them back.